shell/src/main/java/alluxio/shell/command/ReportCommand.java

```diff
@@ -42,7 +44,7 @@ public String getCommandName() {
 
   @Override
   void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    LineageFileSystem.get().reportLostFile(path);
+    LineageFileSystem.get(FileSystemContext.INSTANCE, LineageContext.INSTANCE).reportLostFile(path);
```

Comment:
Not something this PR needs to address, but it seems weird that we directly access `LineageFileSystem` here.